### DISCUSSION TITLE
add Staffmsg option without senders name

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#7937] [Enhancement]
Currently we got the option to send staffsmgs
These are great to send messages to the staff.
But if the sender of this is an automatism it shows the Server as sender. (for example when its made with asConsole!)
It would be nice to have an option like start messsage with ! like in a msg to not show the sender if sender has a permission for it

permission for this: **_cmi.command.staffmsg.clean_**   - allows to send clean message to staffchat (no sendername) if message starts with !

normal staffmsg
![Snag_13300c](https://github.com/Zrips/CMI/assets/12080017/36bd0be0-dccd-4723-b6a0-149b9d9e2fa3)

When using ! in staffmsg
![image](https://github.com/Zrips/CMI/assets/12080017/2b93fbfc-498d-4330-bbac-79ae2c19f0c6)

**Comment by MrCrazys** [comment]
I would love to see this feature implemented. Would be really useful.
